config/initializers/omniauth.rb

```diff
@@ -0,0 +1,5 @@
+OmniAuth.config.logger = Rails.logger
+
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :google_oauth2, ENV['CLIENT_ID'], ENV['CLIENT_SECRET']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.